packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -105,8 +83,8 @@ export function FileUploader({
       const status = fileStatusesRef.current[index];
       fileStatusesRef.current[index] =
         percentage !== 100
-          ? { ...status, percentage, loading: true }
-          : { ...status, percentage, loading: false, success: true };
+          ? { ...status, percentage, fileState: null, isLoading: true }
```

Comment:
The `fileState` string now tracks if the file is in an error, success, or paused state